### PR TITLE
📖  [AMP Story Paywall] amp-story-subscriptions documentation

### PR DESCRIPTION
**Pull request by ychsieh**
The public markdown file in this PR is based on the internal API documentation draft here: [go/amp-story-subscriptions-documentation](https://goto.google.com/amp-story-subscriptions-documentation)

### CODE REVIEW

**Comment by newmuis** [comment]
Is there a reason for this to be internal?  Can you convert it to markdown and include it here?

File: extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md
Comment (processprocess) [inline extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md:117]: Please add what the default would be when optional attributes are not present.

Comment (ychsieh) [reply extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md]: The line would be removed if nothing is provided. No default value here.

**Review by processprocess — APPROVED** [review]
Approving with a nit.
I'll leave it to @CrystalOnScript for the final review!

File: extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md
Comment (CrystalOnScript) [inline extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md:105]: Do we have sizing and format recommendations we could add here?

File: extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md
Comment (CrystalOnScript) [inline extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md:109]:
```suggestion
The index of the page that displays the subscriptions dialog. If not specified, it defaults to 2, which is the 3rd page of the story. Note that this specified page cannot be the first two pages nor the last page of the story.
```

Comment (CrystalOnScript) [reply extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md:109]: Can we explain more what what mean by index? How do they find which index it is or specify it?

File: extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md
Comment (CrystalOnScript) [inline extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md:113]: Are they able to specify if it's per story or a monthly subscription? Is there a limit to the number of characters? Where and how is this displayed?

**Review by CrystalOnScript — CHANGES_REQUESTED** [review]
Overall looks good, but requesting some clarification on the attributes

File: extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md
Comment (ychsieh) [reply extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md]: Added a reference to amp-story document.

File: extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md
Comment (ychsieh) [reply extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md]: Done

File: extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md
Comment (ychsieh) [reply extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md]: Yes any offering. The limits are not about characters but about the view width so it's responsive. This is displayed in the subscriptions dialog. I have attached a sample image of the feature at the very beginning of the doc.

File: extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md
Comment (gmajoulet) [inline extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md:90]: If there is no difference between the configuration for a Story or classic AMP, I think we could call it out specifically (you can port your existing configuration...) and just link to the doc without giving an example.

Comment (ychsieh) [reply extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md]: Sure. Removed this section and call it out in the summary section.

File: extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md
Comment (gmajoulet) [inline extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md:69]: Can we get it from the `amp-story` metadata? https://github.com/ampproject/amphtml/blob/main/extensions/amp-story/amp-story.md#publisher-logo-src-guidelines
Do publishers need to repeat it?

File: extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md
Comment (gmajoulet) [inline extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md:73]: Plz sort by required first, then optional

File: extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md
Comment (gmajoulet) [inline extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md:79]: Please give one or two examples (e.g. `$10` or `10€`) so people see how they can specify currencies

File: extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md
Comment (ychsieh) [reply extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md]: It is from `amp-story`'s attribute. I was trying to express that this `amp-story` attribute is mandatory for this extension to work. But yes, this attribute is already mandatory. We don't need to repeatedly saying that here. Removed this entirely.

File: extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md
Comment (ychsieh) [reply extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md:77]: Done.

File: extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md
Comment (ychsieh) [reply extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md]: Done.

Btw I never noticed that it is 10€ instead of €10. nice to know : )

**Review by gmajoulet — APPROVED** [review]

**Review by Vue-Pu — APPROVED** [review]
Approve merge changes

**Pull request merged**
